packaging/linux/post_install.sh

```diff
@@ -18,20 +20,34 @@ optDeprecated="/opt/keybase/mount-readme"
 chown root:root "$krbin"
 chmod 4755 "$krbin"
 
+disableRedirector="false"
```

Comment:
You could make this a function that returns an actual code, and then it might be more convenient to put it in the `if` expressions below.